data/providers.js

```diff
@@ -25,6 +25,42 @@ const providerSettings = {
       },
     },
   },
+  oci: {
```

Comment:
I believe this file is currently unused so no need to update it.

Reply:
done